docker-compose.yml

```diff
@@ -10,8 +10,8 @@ services:
       PYTHONDONTWRITEBYTECODE: 1
       DJANGO_SETTINGS_MODULE: "pycon.dev_settings"
       SECRET_KEY: "secret"
-    command: ["./manage.py migrate --no-input && ./manage.py runserver 0.0.0.0:8000"]
+    command: ["./manage.py migrate --no-input && ./manage.py runserver 0.0.0.0:8888"]
     volumes:
       - .:/code
     ports:
-      - "8000:8000"
+      - "8888:8888"
```

Comment:
I am not aware about that, but why did you change the port?

Reply:
8000 is the default for a lot of applications - meaning if you work on another project, you'll likely need to bring the other containers down in order to start epcon. This should reduce the chance of a port collision with other projects.

Reply:
Makefile uses 37266 for the same exact reason (port collision)
(37266 is epcon in t9)

Reply:
It used to use that, before the move to docker. I don't mind changing it back to 37266, but it should not stop this PR from being merged.